R/timeseriessummary-data.R

```diff
@@ -85,6 +85,7 @@ parseTSSThresholds <- function(reportData, timezone){
     thresholds[['periods']] <- lapply(thresholds[['periods']], function(p){
       p[['startTime']] <- formatOpenDateLabel(flexibleTimeParse(p[['startTime']], timezone))
       p[['endTime']] <- formatOpenDateLabel(flexibleTimeParse(p[['endTime']], timezone))
+      p <- p[order(p[['startTime']]),]
```

Comment:
Reviewed this from the bottom up for some reason, but I think this will run into the same issue as the curves where the periods of the threshold will be sorted but the thresholds themselves will not be. 

Unfortunately thresholds don't have an overall periodStartTime like curves do which makes this one a bit trickier to sort. Perhaps you could use dplyr or some other method to add a periodStartTime value to each threshold which is equal to the start time of the earliest period of that threshold and then sort based on those? 

I'm not sure if that's the best way to do this or not. Perhaps @lindsaycarr would have some other suggestions tomorrow when she's available?